src/view/screens/Search/Search.tsx

```diff
@@ -370,6 +370,7 @@ function SearchLanguageDropdown({
 
   return (
     <RNPickerSelect
+      placeholder={{}}
```

Comment:
You can use this to specify what the placeholder says, i.e.

```
placeholder={{
  label: _(msg`Select language...`),
  value: '',
}}
```

Reply:
Talked in Slack, we may want one for "No language", but need to do a little more work